### DISCUSSION TITLE
A parallax Rocket for “takeoff”

### DISCUSSION THREAD

**Discussion by Sandile-Dev01** [filamentphp/filament#8374] [Ideas]
![B0241DCB-E2AD-4D01-97B8-9C114AD41E97](https://github.com/filamentphp/filament/assets/105316897/f4d62119-cf1e-4c26-9a7b-753b7f9b5ba2)

- A parallax rocket which reverse takeoff as we scroll would really be great and have more logic than just a car, in my opinion.
- A rocket does take off so having it instead of a car will be intriguing and will provide a user more clarification about the possibilities that Filament will provide as _the sky is the limit._

**Comment by danharrin** [comment]
Yeah, I agree in principle

**Reply by danharrin** [reply]
but also the car at the end does link back to the start where you can see a car bring in "Accelerated"

**Reply by Sandile-Dev01** [reply]
I also concurrently agree with you but that effect is not implemented on mobile devices while mine can also be implemented in them with great ease.🤷🏽‍♂️😊
